client/src/App.css

```diff
@@ -6,7 +6,8 @@ Text: Jost 100
 Links: Jost (bolder)
 
 COLOR PALETTE
-Background Color: #FFF3E0
+Background Color: #FEFBF7
```

Comment:
Nit: If you find yourself changing this value frequently across your entire CSS file, you can use CSS Variables! These are basically variables... but in CSS. Here's how you might use them:

```
:root: {
  --border-color: #FEFBF7;
}

.recipe-body-left {
    border-top: 1px solid var(--border-color);
}
```

That way, you only have to change them once!

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

These are pretty cool, but definitely don't feel compelled to add them!

Reply:
Yay, this is super useful. Thank you!